strings/base_coroutine_threadpool.h

```diff
@@ -528,18 +556,35 @@ WINRT_EXPORT namespace winrt
         return awaitable{ handle, timeout };
     }
 
+    // HACK: GCC does not compile co_await on thread_pool because it wants
```

Comment:
This is just a helper - how about we just exclude it for GCC? Not loving all the hackery to get GCC to work...

Reply:
Okay, I can revert this one and just exclude the `thread_pool` tests from the GCC build.